index.js

```diff
@@ -7,12 +10,28 @@ function ask(questionText) {
   });
 }
 
+//----------------async code---------------//
+
```

Comment:
Hoisting only works for a certain type of function (declaration) and not the other (expression). While it is viable to hoist your function declaration invocation (as you did here), I would encourage you to not utilize hoisting until you are farther along in your JS journey and the choice is more informed.